oshi-core/src/main/java/oshi/software/os/unix/aix/AixOperatingSystem.java

```diff
@@ -0,0 +1,268 @@
+    @Override
+    public long getSystemUptime() {
+        return System.currentTimeMillis() / 1000L - BOOTTIME;
+    }
+
+    @Override
+    public long getSystemBootTime() {
+        return BOOTTIME;
+    }
+
+    private static long querySystemBootTime() {
+        return Who.queryBootTime() / 1000L;
+    }
+
+    @Override
+    public NetworkParams getNetworkParams() {
+        return new AixNetworkParams();
+    }
+
+    @Override
+    public OSService[] getServices() {
+        // Need to update for whatever services AIX uses
+        List<OSService> services = new ArrayList<>();
+        // Get system services from lssrc command
+        /*-
+         Output:
+         Subsystem         Group            PID          Status
+            platform_agent                    2949214      active
+            cimsys                            2490590      active
+            snmpd            tcpip            2883698      active
+            syslogd          ras              2359466      active
+            sendmail         mail             3145828      active
+            portmap          portmap          2818188      active
+            inetd            tcpip            2752656      active
+            lpd              spooler                       inoperative
+                        ...
+         */
+        List<String> systemServicesInfoList = ExecutingCommand.runNative("lssrc -a");
+        if (systemServicesInfoList.size() > 1) {
+            systemServicesInfoList.remove(0); // remove header
+            for (String systemService : systemServicesInfoList) {
+                String[] serviceSplit = ParseUtil.whitespaces.split(systemService.trim());
+                if (systemService.contains("active")) {
+                    if (serviceSplit.length == 4) {
+                        services.add(new OSService(serviceSplit[0], ParseUtil.parseIntOrDefault(serviceSplit[2], 0),
+                                RUNNING));
+                    } else if (serviceSplit.length == 3) {
+                        services.add(new OSService(serviceSplit[0], ParseUtil.parseIntOrDefault(serviceSplit[1], 0),
+                                RUNNING));
+                    }
+                } else if (systemService.contains("inoperative")) {
+                    services.add(new OSService(serviceSplit[0], 0, STOPPED));
+                }
+            }
+        }
+        // Get installed services from /etc/rc.d/init.d
+        File dir = new File("/etc/rc.d/init.d");
+        File[] listFiles;
+        if (dir.exists() && dir.isDirectory() && (listFiles = dir.listFiles()) != null) {
+            for (File file : listFiles) {
+                String installedService = ExecutingCommand.getFirstAnswer(file.getAbsolutePath() + " status");
+                // Apache httpd daemon is running with PID 3997858.
+                if (installedService.contains("running")) {
+                    Matcher m = ParseUtil.AIX_RUNNING_SERVICE_INFO.matcher(installedService);
```

Comment:
This can be greatly simplified with `ParseUtil.parseLastInt()` to grab the PID.

Reply:
done.

Reply:
You might also want to delete the regexp constant from the `ParseUtil`.